saleor/core/utils/promo_code.py

```diff
@@ -9,14 +9,19 @@
 class InvalidPromoCode(ValidationError):
     def __init__(self, message=None, **kwargs):
         if message is None:
-            message = {"promo_code": "Promo code is invalid"}
+            message = {
+                "promo_code": ValidationError("Promo code is invalid", code="invalid")
```

Comment:
Shouldn't we use `ErrorCode` class for `code` here?

Reply:
I've added an import from `GiftCardError` here; PR for changing that: #4744